src/components/user/UserProfile.js

```diff
@@ -79,6 +114,54 @@ export default function PersonalProfile() {
           View Recommendations
         </Button>
       </Link>
+
+      <h3 className="text-uppercase">Purchased Albums</h3>
+      {error && <Alert severity="error">{error}</Alert>}
+
+      {loading && <p>Loading purchased albums...</p>}
+      <Table className="text-center mt-5">
+        <tbody>
+          <TableRow>
+            <TableCell>
+              {' '}
+              <b>
+                {' '}
+                <em></em>
+              </b>
+            </TableCell>
```

Comment:
We could just have an empty `TableCell` component here to represent the empty header for this column:
```suggestion
            <TableCell/>
```